plugins/modules/elasticsearch_plugin.py

```diff
@@ -293,7 +298,8 @@ def main():
         module.exit_json(changed=False, name=name, state=state)
 
     if state == "present":
-        changed, cmd, out, err = install_plugin(module, plugin_bin, name, version, src, url, proxy_host, proxy_port, timeout, force)
+        changed, cmd, out, err = install_plugin(module, plugin_bin, name, version, src, url, proxy_host, proxy_port,
+                                                timeout, force)
```

Comment:
This change, and the other one above, seem to be just reformatting - for 80 columns maybe? In the phone right now, hard to tell. Anyways, the adopted standard for thw collection is 160 chars per line, so these changes are kinda redundant and bound to be reverted later by someone else.

Reply:
Ah my bad, I just reverted these changes.